corehq/apps/cloudcare/templates/form_entry/templates.html

```diff
@@ -29,9 +31,10 @@
               },
         click: toggleChildren,
         event: {keypress: keyPressAction},
-        ">
+        style: {'background-color': headerBackgroundColor(),
+                'color': headerBackgroundColor() ? 'white' : '',
+        }">
       <div data-bind="ifnot: collapsible">
-        <legend>
```

Comment:
I think `legend` here makes this text correctly label the `fieldset` that this group is, which could be important for accessibility.

Reply:
@nospame @stephherbers how would I verify that this is not breaking any accessibility?

Reply:
You should be able to check with a screen reader or directly in browser accessibility tools. When the item this is labeling is selected (the group itself, I believe), the label should be present. In Chrome this looks something like "From legend: `legend.sr-only`". However after navigating through and checking this out myself, I think it's not actually effectively labeling anything, since every input has its own label, though it's also probably not doing any harm. Possibly something I can look into as part of USH-4106.